recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
 {% set name = "jupyterlab_widgets" %}
-{% set version = "3.0.9" %}
+{% set version = "3.0.10" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab_widgets-{{ version }}.tar.gz
-  sha256: 6005a4e974c7beee84060fdfba341a3218495046de8ae3ec64888e5fe19fdb4c
+  sha256: 04f2ac04976727e4f9d0fa91cdc2f1ab860f965e504c29dbd6a65c882c9d04c0
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  skip: True # [py<37]
+  skip: True # [py<37 or s390x]
 
 requirements:
   host:
```

Comment:
I think this bound can be increased to <2 as its listed [here](https://github.com/jupyter-widgets/ipywidgets/blob/main/python/jupyterlab_widgets/pyproject.toml#L2).

`- jupyter-packaging >=0.10,<2`

Reply:
updated, thank you